app/src/main/java/com/github/odaridavid/weatherapp/util/NotificationUtil.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.odaridavid.weatherapp.util
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import com.github.odaridavid.weatherapp.R
+import javax.inject.Inject
+
+class NotificationUtil @Inject constructor(
+    private val context: Context
+) {
+    fun makeNotification(message: String) {
+        createNotificationChannel()
+
+        val builder = NotificationCompat.Builder(context, "default")
+            .setSmallIcon(R.drawable.ic_launcher_foreground)
+            .setContentText(message)
+            .setPriority(NotificationCompat.PRIORITY_DEFAULT)
+    }
+
+    private fun createNotificationChannel() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            val name = "notification"
+            val descriptionText = "description"
```

Comment:
I'd suggest more meaningful names and description on what kind of notificiations this channel provides e.g Weather Updates or something in that line

Reply:
okay.